website/src/docs/experiments/remote-taskfiles.md

```diff
@@ -34,19 +36,35 @@ when including Taskfiles.
 Task uses "nodes" to reference remote Taskfiles. There are a few different types
 of node which you can use:
 
-<Tabs groupId="method" queryString>
-<TabItem value="http" label="HTTP/HTTPS">
+::: code-group
+
+```text [HTTP/HTTPS]
+https://raw.githubusercontent.com/go-task/task/main/website/static/Taskfile.yml
+```
+
+```text [Git over HTTP]
+https://github.com/go-task/task.git//website/static/Taskfile.yml?ref=main
+```
+
+```text [Git over SSH]
+git@github.com/go-task/task.git//website/static/Taskfile.yml?ref=main
+```
+
+:::
```

Comment:
Kind of a shame that we lose the id/anchor features from Docusaurus here.

i.e. https://taskfile.dev/experiments/remote-taskfiles/?method=git-ssh will take you to the page with the tab set to the given method and changing the method updates every set of tabs on the page.

Maybe this is a feature request we can make and re-add the functionality later

Reply:
Should I sleep in the night or develop this feature for vitepress ? do not tempt me :joy: